packages/ubuntu_wizard/lib/src/wizard_button.dart

```diff
@@ -228,7 +228,10 @@ class NextWizardButton extends StatelessWidget {
   final bool highlighted;
   final Object? arguments;
   final WizardCallback? onNext;
-  final WizardCallback? onExecute;
+
+  /// Called when returning to the current route after navigating to the next one.
+  /// (e.g. when pressing the back button on the next page)
+  final WizardCallback? onReturn;
```

Comment:
It's weird that the button is responsible for this... But that is for a later refactor.

Reply:
Yes, it's quite ugly too carry around a chain of coroutines like this - and terribly annoying to debug